packages/jest-runtime/src/__tests__/runtime_require_module.test.js

```diff
@@ -9,6 +9,7 @@
 'use strict';
 
 import path from 'path';
+// eslint-disable-next-line import/default
```

Comment:
can we set this rule for `.ts` files only instead of disabling in JS?

Reply:
We disable it in ts already (as `tsc` will yell at us).